### PR TITLE
Fixed wrong calculation of estimated reading time

### PR DESCRIPTION
**Pull request by SorenThorup**

### CODE REVIEW

File: src/Actions/Backend/EstimatedReadingTime.php [hunk 1]
```diff
@@ -26,20 +25,21 @@ public function addEstimatedReadingTime($postId) {
                 case 'file':
                     break;
                 case 'gallery':
+                    var_dump($item->getImages());
```
Comment (MFlor) [inline src/Actions/Backend/EstimatedReadingTime.php]: Rogue `var_dump`

File: src/Actions/Backend/EstimatedReadingTime.php [hunk 2]
```diff
@@ -84,7 +86,12 @@ public function addTimeForImages($amountOfImages)
         $initialSecondsPerImage = 12;
 
         for($i=0; $i<$amountOfImages; $i++) {
-            $seconds = $seconds + ($initialSecondsPerImage - $i);
+            if($i<10) {
+                $seconds = $seconds + ($initialSecondsPerImage - $i);
+            } else {
+                $seconds = $seconds + 3;
+            }
```
Comment (MFlor) [inline src/Actions/Backend/EstimatedReadingTime.php]: Maybe add some comments on why you are adding three here and the general calculation

File: src/Actions/Backend/EstimatedReadingTime.php (hunk 1 above)
Comment (SorenThorup) [reply src/Actions/Backend/EstimatedReadingTime.php]: OMG!!!

File: src/Actions/Backend/EstimatedReadingTime.php (hunk 2 above)
Comment (SorenThorup) [reply src/Actions/Backend/EstimatedReadingTime.php]: Ofc - Done!

**Pull request merged**